lib/Conversion/TorchToTcp/DataMovement.cpp

```diff
@@ -26,6 +28,75 @@ using namespace mlir::torch;
 using namespace mlir::torch::Torch;
 
 namespace {
+template <typename OpTy, typename OpAdaptor>
```

Comment:
Do you expect this to be useful for ops other than `AtenSliceTensorOp`? Just wondering if there's a need to templatize.

Reply:
Not for now. This part of code is copied from [here](https://sourcegraph.com/github.com/llvm/torch-mlir@d65925a8b465d4a84be947d37197178d5c5cc6d2/-/blob/lib/Conversion/TorchToLinalg/DataMovement.cpp?L42). Will add a note about it.